qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -104,4 +105,26 @@
       <image>https://organicmaps.app/images/screenshots/dark.jpg</image>
     </screenshot>
   </screenshots>
+
+  <releases>
+   <release  version="2022.03.23-4-android" date="2022-03-23">
```

Comment:
Is this the correct version name? Or should it be just "2022.03.23-4"?

Reply:
Is it an abstract version or a reference to the tag in the repo?

Reply:
It's an abstract version

Reply:
Then you can remove the -android part.

Reply:
Done